tools/check-branch

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+if ! git diff-index --quiet HEAD; then
+    set +x
+    echo "There are uncommitted changes:"
```

Comment:
I'm not sure this works because I had uncommitted changes and it still started testing.

Reply:
Could you give more information about this situation? `tools/fetch-pull-request` does something similar and I took the test from there. It seems to work for me with simple testing.